### ISSUE TITLE
stop print opens web site

### ISSUE THREAD

**Issue by UBaer21** [bambulab/BambuStudio#5319]
### Bambu Studio Version

1.10.1.50

### Where is the application from?

Bambu Lab Official website

### OS version

windows 10

### Printer

P1S

### How to reproduce

![image](https://github.com/user-attachments/assets/004c1b08-1f3e-4842-aca8-a59b123f125f)
![image](https://github.com/user-attachments/assets/cd45e4c6-e166-4ae0-86a4-5babe3592f1b)
YES

### Actual results

opens Browser 
https://wiki.bambulab.com/en/software/bambu-studio/faq/live-view

### Expected results

Not open Browser website

### Project file & Debug log uploads

not needed

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by JohnSmithCYM** [comment]
It is a mistake when your pop-up button is laid on the top of liveview error string, move the pop-up to left or right a little can resolve this question, I believe they will fix that next version.

**Comment by UBaer21** [comment]
@JohnSmithCYM you are right that this is a click through, although the link below is not visible as a link 
![image](https://github.com/user-attachments/assets/7f9d01c7-7a7d-45ad-91b8-6d4007723102)

They should not allow a click through to the underlaying element.
